tools/process_symbol_map.py

```diff
@@ -93,15 +93,35 @@ class Map:
     include_list = []
     var_count = 0
 
+    # Info for error logging
+    current_file = ""
+    current_file_issues = []
+
     def parse(self, file):
-        reader = json.load(file)
-        for key in reader:
-            self.__parse(key, reader[key])
+        self.current_file = file.name
+        self.current_file_issues = []
+        
+        try:
+            reader = json.load(file)
+            for key in reader:
+                self.__parse(key, reader[key])
+
+        except json.decoder.JSONDecodeError:
+            print(f"Cannot parse '{file.name}', file does not contain valid JSON")
+            return
+        
+        # Don't print any info for succesfully parsed files
+        if len(self.current_file_issues) == 0:
+            return
+            
+        print(f"Found {len(self.current_file_issues)} issue{'' if len(self.current_file_issues) == 1 else 's'} in {self.current_file}")
```

Comment:
> I also want to take a look at how this looks when printing so I'll need to try it out before merging, otherwise looks fun

![image](https://user-images.githubusercontent.com/69014593/234397104-1a06ec22-adbc-4a8c-bfe3-59cdb9c30a4e.png)